vllm/datadog_checks/vllm/metrics.py

```diff
@@ -0,0 +1,39 @@
+# (C) Datadog, Inc. 2024-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+METRIC_MAP = {
```

Comment:
nit. What is the criteria that is used for mapping metrics here? 

```
   'vllm:cache_config_info': 'cache_config_info',
    'vllm:gpu_cache_usage_perc': 'cache_usage_perc',
    'vllm:cpu_cache_usage_perc': 'cpu_cache_usage_perc',
```

Reply:
I've thought about this a bit and pushed something that I think is clearer. lmk